subsys/net/openthread/Kconfig.defconfig

```diff
@@ -0,0 +1,149 @@
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# The purpose of this file is to define new default values of settings related
+# to building Thread from sources.
+# This file only changes defaults and thus all symbols here must be prompltless
+# and safeguarded so that they only are applied when building Thread.
+
+if NET_L2_OPENTHREAD
+
+choice LIBC_IMPLEMENTATION
+	default NEWLIB_LIBC
+endchoice
+
+config NUM_METAIRQ_PRIORITIES
+	int
+	default 1
+
+config SYSTEM_WORKQUEUE_STACK_SIZE
+	int
+	default 1120
+
+config HW_STACK_PROTECTION
+	bool
+	default y
+
+config NET_CONNECTION_MANAGER
+	bool
+	default y
+
+config MAIN_STACK_SIZE
+	int
+	default 2048
+
+config INIT_STACKS
+	bool
+	default y
+
+config NUM_METAIRQ_PRIORITIES
+	int
+	default 1
+
+config NET_LOG
+	bool
+	default y
+
+config NET_STATISTICS
+	bool
+	default y
+
+config NET_IPV6_NBR_CACHE
+	bool
+	default n
+
+config NET_IPV6_MLD
+	bool
+	default n
+
+config NET_TX_STACK_SIZE
+	int
+	default 1200
```

Comment:
Already default to 1200 in Zephyr, why is it needed here ?
https://github.com/nrfconnect/sdk-zephyr/blob/66ef6a7e125d53b5d650effcce101ab901627100/subsys/net/ip/Kconfig.stack#L10
```
config NET_TX_STACK_SIZE
	int "TX thread stack size"
	default 1200
```